lib/client/player.go

```diff
@@ -50,7 +57,8 @@ type sessionPlayer struct {
 	sessionEvents []events.EventFields
 	term          *terminal.Terminal
 
-	// stopC is used to tell the caller that player has finished playing
+	// stopC is called when playback ends (either because the end of the stream has
```

Comment:
"called" or "closed"? If the latter, then:

```suggestion
	// stopC is closed when playback ends (either because the end of the stream has
```